firedrake/cython/dmcommon.pyx

```diff
@@ -2298,16 +2308,17 @@ def orientations_facet2cell(
                 dst_orient[i] = (cone_orient[i] < 0) ^ facet_orientations[cone[i] - fStart]
 
             # We select vertex X (figure above) as starting vertex.
-            # Both traversal order (CCW or CW) is fine. We choose the traversal
-            # where the second vertex has the smaller global number.
+            # We choose the traversal where the X and the second vertex
+            # would make either:
+            # - the cone of the first facet, or
+            # - the cone of the opposing facet of the first facet
+            # where the facet ordering is given by the plex cell cone.
             #
-            # The other traversal other would be an equally good choice,
-            # however, for cells in the halo, the same choice must be made in
-            # each MPI process which sees that cell.
+            # This ensures that cells in the halo are consistently
+            # oriented across MPI processes.
             #
-            # To ensure this, we only calculate cell orientations for the
-            # locally owned cells, and later exchange these values on the
-            # halo cells.
+            # We only calculate cell orientations for the locally owned
+            # cells, and later exchange these values on the halo cells.
```

Comment:
These two changed comments seem to be in conflict. The first says:

> We ensure cells in the halo are consistently oriented

The second contradicts this with:

> We only calculate for locally owned and then exchange in the halo.

Which is it?